api/api/spec/spec.yaml

```diff
@@ -1969,6 +1951,65 @@ components:
               - "yes"
               - "no"
 
+    # KVDB models
+    ItemKVDB:
+      allOf:
+        - $ref: '#/components/schemas/CommonRuleset'
+        - type: object
+          description: "KVDB asset descriptor and content"
+          required:
+            - type
+            - id
+            - name
+            - content
+          properties:
+            type:
+              type: string
+              description: "Asset type. Always 'kvdb'."
+              enum:
+                - kvdb
```

Comment:
I know this comes from the endpoint definition, but if it will be always be `kvdb`, what's the point of having the field? Will we add more values in the future? (Would be weird given that the endpoints are called `kvdb`) 

If not, I would contact the other divisions to try and get rid of it.